rebar.config

```diff
@@ -29,7 +29,8 @@
 {deps, [
         {leo_commons, ".*", {git, "https://github.com/leo-project/leo_commons.git", {tag, "1.1.1"}}},
         {leo_mcerl,   ".*", {git, "https://github.com/leo-project/leo_mcerl.git",   {branch, "develop"}}},
-        {leo_dcerl,   ".*", {git, "https://github.com/leo-project/leo_dcerl.git",   {branch, "develop"}}}
+%        {leo_dcerl,   ".*", {git, "https://github.com/leo-project/leo_dcerl.git",   {branch, "develop"}}}
+        {leo_dcerl,   ".*", {git, "https://github.com/windkit/leo_dcerl.git",       {branch, "cache_tran"}}}
```

Comment:
@windkit You could use leo_dcerl's develop branch because I've merged your request with it.